R/plot_convehull.r

```diff
@@ -0,0 +1,108 @@
+#' Function for plotting convex hulls.
```

Comment:
please rename this to `plot_convexhull.r` :)

Reply:
Ohh lol sure. Do I create a new pull request?

Reply:
no you can just delete the file and commit again, it will update this PR 

you can keep committing while the PR is open